### PR TITLE
Added a hotkey to go to the home page

### PR DESCRIPTION
**Pull request by omertuc**
1. LINKED ISSUES:
None

2. TEST URLS:
None

3. SCREENSHOT:
None

Added a `g` `h` shortcut to go to the homepage of GitHub.

Copied most of it from the `profile-hotkey` feature.

Having `g` `h` as a keyboard shortcut to go to the homepage appears to be quite common:
- One of my extensions does it (not sure which) on Stack Overflow
- Twitter does it natively

I got used to it and felt like GitHub should have it too.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -289,6 +289,7 @@ Thanks for contributing! 🦋🙌
 - [](# "format-conversation-titles") [Makes issue/PR references in issue/PR titles clickable and renders `` `text in backticks` ``.](https://user-images.githubusercontent.com/22439276/58927232-71ae2780-876b-11e9-941e-bb56a7389123.png)
 - [](# "trending-menu-item") Adds a `Trending` link to the global navbar and a keyboard shortcut: <kbd>g</kbd> <kbd>t</kbd>.
 - [](# "navigate-pages-with-arrow-keys") Adds shortcuts to navigate through pages with pagination: <kbd>←</kbd> and <kbd>→</kbd>.
+- [](# "home-hotkey") Adds a keyboard shortcut to visit the GitHub home page: <kbd>g</kbd> <kbd>h</kbd>.
```
Comment (yakov116) [inline readme.md:292]: Move to the bottom of the list

File: source/refined-github.ts [hunk 2]
```diff
@@ -44,6 +44,7 @@ import './features/edit-comments-faster';
 import './features/open-all-notifications';
 import './features/copy-on-y';
 import './features/profile-hotkey';
+import './features/home-hotkey';
```
Comment (yakov116) [inline source/refined-github.ts:47]: Move to the bottom of the list

File: source/features/home-hotkey.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'dom-chef';
+import onetime from 'onetime';
+import {isEnterprise} from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	const homepageLink = (isEnterprise() ? location.origin : 'https://github.com');
```
Comment (yakov116) [inline source/features/home-hotkey.tsx:8]:
```suggestion
	const homepageLink = isEnterprise() ? location.origin : 'https://github.com';
```

**Review by yakov116 — CHANGES_REQUESTED** [review]

File: source/features/home-hotkey.tsx (hunk 3 above)
Comment (fregante) [reply source/features/home-hotkey.tsx:8]: They are identical, there should not a be a condition at all. It's always `location.origin` and it should just be inlined in the JSX.

**Comment by fregante** [comment]
Thanks for the PR, but this already exists natively:  
Screen Shot 2020-12-14 at 19 51 43

**Pull request closed without merging**